### PR TITLE
IMED-02: Created Drug List Fragment

### PR DESCRIPTION
**Pull request by Saotomefirst**
**Changes:**

- Created Drug List RecyclerView
- loaded mock list into the RecyclerView
- Renamed FirstFragment to DrugListFragment
- Moved MainActivity into ui package and Fragments into ui.fragments package

image

### CODE REVIEW

File: app/src/main/java/com/saotome/imed_app/ui/fragments/DrugListAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            binding.tvStartTime.text = item.startTime
+        }
+    }
+}
+
+class DiffCallback : DiffUtil.ItemCallback<Drug> () {
+    override fun areItemsTheSame(oldItem: Drug, newItem: Drug): Boolean = oldItem == newItem
+
+    override fun areContentsTheSame(oldItem: Drug, newItem: Drug): Boolean = oldItem.id == newItem.id
+
```
Comment (maylcf) [inline app/src/main/java/com/saotome/imed_app/ui/fragments/DrugListAdapter.kt]: Remove this line

File: app/src/main/java/com/saotome/imed_app/ui/fragments/DrugListFragment.kt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import com.saotome.imed_app.ui.DrugListAdapter
+
+/**
+ * A simple [Fragment] subclass as the default destination in the navigation.
+ */
+class DrugListFragment : Fragment() {
+
+    private var _binding: FragmentDrugListBinding? = null
+    private val drugList = arrayListOf<Drug>()
+    // This property is only valid between onCreateView and
```
Comment (maylcf) [inline app/src/main/java/com/saotome/imed_app/ui/fragments/DrugListFragment.kt]: Does this comment still make sense? I think you removed the line below it.

File: app/src/main/java/com/saotome/imed_app/ui/fragments/DrugListAdapter.kt (hunk 1 above)
Comment (Saotomefirst) [reply app/src/main/java/com/saotome/imed_app/ui/fragments/DrugListAdapter.kt]: Done
ac033076add9121739da75bacc1053902964c6a8

File: app/src/main/java/com/saotome/imed_app/ui/fragments/DrugListFragment.kt (hunk 2 above)
Comment (Saotomefirst) [reply app/src/main/java/com/saotome/imed_app/ui/fragments/DrugListFragment.kt]: No it doesn't. Thanks for the reminder :)
ac033076add9121739da75bacc1053902964c6a8

**Review by maylcf — APPROVED** [review]
👍

**Pull request merged**